### DISCUSSION TITLE
"wrong version number"

### DISCUSSION THREAD

**Discussion by plancked** [postalserver/postal#3205] [Help with using Postal] [answered]
Hi,

I'm trying to use GrooveHQ with my postal smtp service.
When trying to connect it reports:

`SSL_connect returned=1 errno=0 peeraddr=103.**.***.**:25 state=error: wrong version number`

It works with other services (postmark, send grid)

Any ideas?

thanks
Alex

**Comment by willpower232** [comment]
What console command did you run to get that?

**Comment by plancked** [comment]
That error is reported my groovehq, when it tries to connect to Postal.

**Reply by willpower232** [reply, marked as answer]
I think you need to untick "Connect using SSL", I don't think GrooveHQ supports STARTTLS which is the only SSL Postal supports.
